offline-status/index.js

```diff
@@ -0,0 +1,32 @@
+// Register the ServiceWorker
+navigator.serviceWorker.register('service-worker.js', {
+  scope: '.'
+}).then(function(registration) {
+  console.log('The service worker has been registered ', registration);
+});
+
+// Listen for claiming of our ServiceWorker
+navigator.serviceWorker.addEventListener('controllerchange', function(event) {
+  console.log('[controllerchange] A "controllerchange" event has happened within navigator.serviceWorker: ', event);
+
```

Comment:
Should we do some sort of `scriptURL` validation here to ensure it's the correct service worker?